<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIApp</class>
 <widget class="QMainWindow" name="GUIApp">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Denis PC Hardware Shop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QComboBox" name="itemComboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QFrame" name="CPUInput">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>251</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="CPUNameL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CPUName">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="CPUPriceL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>49</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CPUPrice">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="CPUBrandL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Brand</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CPUBrand">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="CPUCoreL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Core count</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="CPUCore">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>42</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>32</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="CPUFrequency">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>62</width>
       <height>25</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="CPUFrequencyL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Frequency</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CPUAdd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>221</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Add/Modify CPU</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CPUID">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="CPUIDL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>49</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="RemoveButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RemoveID">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="FilterTypeButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter by type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FilterPriceButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter by price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FilterType">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FilterPrice">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PrintAllButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Display All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UndoButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RedoButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Redo</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ItemList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>541</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Denis PC Hardware Store Inventory Management System</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>578</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
